### DISCUSSION TITLE
Can't download likes (own or others)

### DISCUSSION THREAD

**Discussion by ZeroThreeQuarter** [TumblThreeApp/TumblThree#332] [Q&A (Question & Answer)]
I'm clearly doing something wrong where i can't download my own likes, no anyone elses.

I've authenticated my account and all, but when i click "download" absolutely nothing is happening in the interface.

Any help would be appreciated, i've only read good things about this tool, so i'm bummed it's not working.

Thanks all!

**Comment by thomas694** [comment]
The download of Likes has currently no problems.

So we need to find out why it's not working to you. A couple of things you can check. It can take quite a few seconds until the download starts. Have you added the blog to the Queue tab by e.g. double clicking it? How does it show in the Queue tab after you pressed Download, any status messages under the blog name?
Do you have any hidden blogs on your blog list? Can you download them? Anyway, you can just try to log out and in again.
You can also have a look in the log file (%LOCALAPPDATA%\TumblThree\TumblThree.log), whether there's an error at the time you try to download.
